src/main/java/com/michelin/ns4kafka/service/executor/AccessControlEntryAsyncExecutor.java

```diff
@@ -189,7 +180,8 @@ private List<AclBinding> collectBrokerAcls(boolean managedUsersOnly)
             throws ExecutionException, InterruptedException, TimeoutException {
         List<ResourceType> validResourceTypes = List.of(
                 org.apache.kafka.common.resource.ResourceType.TOPIC,
-                org.apache.kafka.common.resource.ResourceType.GROUP);
+                org.apache.kafka.common.resource.ResourceType.GROUP,
+                org.apache.kafka.common.resource.ResourceType.TRANSACTIONAL_ID);
```

Comment:
With transaction ACL being retrieved from the broker and removed from the `buildAclBindingsFromKafkaStream`, all Kafka Streams transaction ACL will be dropped.